Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1253,9 +1535,7 @@ def list_scripts_command():
     human_readable = tableToMarkdown('CrowdStrike Falcon scripts', scripts)
 
     entry_context = {
-        'CrowdStrike': {
```

Comment:
while this change makes sense, it break bc
need to document that in the release notes